handlers/graph.go

```diff
@@ -81,9 +81,15 @@ func graphNamespaces(o options.Options, client *prometheus.Client) graph.Traffic
 
 	trafficMap := graph.NewTrafficMap()
 	for _, namespace := range o.Namespaces {
+		// If we don't have access to the namespaces being requested then return an error.
+		if _,found := o.AccessibleNamespaces[namespace]; !found {
+			checkError(errors.New(fmt.Sprintf("Kiali does not have access to the %v namespace", namespace)))
+		}
 		log.Debugf("Build traffic map for namespace [%s]", namespace)
 		namespaceTrafficMap := buildNamespaceTrafficMap(namespace, o, client)
 
+                markInaccessible(namespaceTrafficMap, o)
```

Comment:
I don't think you want to do this here.  You'd be better off doing this in the markOutsiders logic.  You want to wait for all of the appenders to run first.  Also, only outsider nodes can be inaccessible.

Reply:
The problem is that we can't wait for all the appenders to run first, some of the appenders will fail if it ties to access information in a namespace where it doesn't have access (eg the sidecar check appender)